builder-support/helpers/install_quiche.sh

```diff
@@ -55,7 +55,7 @@ install -m644 quiche/include/quiche.h "${INSTALL_PREFIX}"/include
 install -m644 target/release/libquiche.${SOEXT} "${LIBDIR}"/libdnsdist-quiche.${SOEXT}
 
 if [ $(uname) = Darwin ]; then
-  install_name_tool -id "${LIBDIR}/libdnsdist-quiche.${SOEXT}" "${LIBDIR}"libdnsdist-quiche.${SOEXT}
+  install_name_tool -id "${LIBDIR}/libdnsdist-quiche.${SOEXT}" "${LIBDIR}/"libdnsdist-quiche.${SOEXT}
```

Comment:
For consistency with line 55 this should be:
```suggestion
  install_name_tool -id "${LIBDIR}/libdnsdist-quiche.${SOEXT}" "${LIBDIR}"/libdnsdist-quiche.${SOEXT}
```